app/src/main/res/values/styles.xml

```diff
@@ -115,6 +115,107 @@
         <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
     </style>
 
+    <!-- DrawerArrowStyle is used to change the color of hamburger-icon to white -->
+    <style name="DrawerArrowStyle" parent="Widget.AppCompat.DrawerArrowToggle">
+        <item name="color">@android:color/white</item>
+    </style>
+
+    <style name="ToolbarTheme" parent="ThemeOverlay.AppCompat.ActionBar">
+        <item name="drawerArrowStyle">@style/DrawerArrowStyle</item>
+        <item name="colorControlNormal">@color/white</item>
+
+        <item name="thumbs_up">@drawable/ic_thumb_up_white_24dp</item>
```

Comment:
Now the build is failing because some icon attrs were renamed (e.g. thumbs_up is now ic_thumbs_up). So please copy-paste this list of attributes again

Reply:
Done. All checks passing.
Thanks